src/intro.md

```diff
@@ -6,46 +6,43 @@ The Internet currently lacks effective, privacy-preserving **Data Provenance**.
 
 Enter TLSNotary: a protocol enabling users to export data securely from any website. Using Zero Knowledge Proof (ZKP) technology, this data can be selectively shared with others in a cryptographically verifiable manner.
 
-TLSNotary makes data truly portable and allows users to share it with others as they see fit.
+TLSNotary makes data truly portable and allows a user, the `Prover`, to share it with another party, the `Verifier`, as they see fit.
 
 ## How Does the TLSNotary Protocol Work?
 
-The TLSNotary protocol consists of 4 steps:
-1. The `User` **requests** the data from the `Server` privately and securely.
-2. The `Notary` **checks** for data tampering and **signs** the data.
-3. The `User` **selectively discloses** the data to the `Verifier`.
-4. The `Verifier` **verifies** the data.
+The TLSNotary protocol consists of 3 steps:
+1. The `Prover` **requests** data from a `Server`. The `Verifier` participates with secure and privacy-preserving *multi party computation (MPC)*.
+2. The `Prover` **selectively discloses** the data to the `Verifier`.
+3. The `Verifier` **verifies** the data.
 
-![](./png-diagrams/overview3.png)
+![](./png-diagrams/overview4.png)
 
 ### ① Multi-party TLS Request
 
-TLSNotary works by adding a third party, the `Notary`, to the usual TLS connection between the `User` and a `Server`. This `Notary` is **not "[a man in the middle](https://en.wikipedia.org/wiki/Man-in-the-middle_attack)"**. Instead, the `Notary` participates in a **secure multi-party computation** (MPC) to jointly operate the TLS connection without ever seeing the data in plain text; the `Notary` only sees encrypted data. Given that the `Notary` only sees the temporary key of the `Server`, the `Notary` does not know which `Server` the `User` is communicating with. The TLSNotary protocol is transparent to the `Server`. From the `Server`'s perspective, the `User`'s connection is a standard TLS connection.
+TLSNotary works by adding a third party, a `Verifier`, to the usual TLS connection between the `Prover` and a `Server`. This `Verifier` is **not "[a man in the middle](https://en.wikipedia.org/wiki/Man-in-the-middle_attack)"**. Instead, the `Verifier` participates in a **secure multi-party computation** (MPC) to jointly operate the TLS connection without seeing the data in plain text. By participating in the MPC, the `Verifier` can validate the authenticity and integrity of the data the `Prover` received from the `Server`.
```

Comment:
The answer here https://security.stackexchange.com/questions/93322/difference-between-authentication-integrity-and-data-origin-authentication
explains that TLS data authenticity automatically implies "integrity", so we can drop integrity, imo